### PR TITLE
pages functions billing info

### PR DESCRIPTION
**Pull request by nevikashah**

### CODE REVIEW

**Comment by WalshyDev** [comment]
gonna ping @Cherry for a quick look

File: content/pages/platform/functions/billing.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## About Pages Functions Billing 
+One of the key features of Pages Functions is limitless requests. In beta, Pages Functions had a 100,000 requests/day limit. The Pages Functions billing model introduced on November 15th allows your traffic to scale without limitation.
+
+## How it works 
+Pages Functions billing will be merged with the existing Workers billing structure for Free, Paid, and Enterprise customers. Read below to learn what this means for you as it relates to your Workers subscription plan: 
+
+### Workers Paid 
+* If you currently have a Workers Paid subscription, billing will automatically occur for all Functions usage after November 15th –- no additional action required. Pages Functions will share the same pool of quotas included in the subscription (for example, requests and duration).
+* For example, you will share the included 10 million Bundled requests between Workers and Pages Functions.
```
Comment (WalshyDev) [inline content/pages/platform/functions/billing.md]:
```suggestion
  * For example, you will share the included 10 million Bundled requests between Workers and Pages Functions.
```

**Review by WalshyDev — APPROVED** [review]
LGTM - PCX review happened off docs

**Review by Cherry — COMMENTED** [review]
This looks great - excited for this to land 👀 Thanks for the ping.

One suggestion: including a note here for users who had their functions limit increased beyond 100k during beta might be a good idea. Something to make it clear that their increased limits during beta no longer apply and anything above 100k will be billed.

File: content/pages/platform/functions/billing.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+# Pages Functions GA Billing 
+
+## About Pages Functions Billing 
+One of the key features of Pages Functions is limitless requests. In beta, Pages Functions had a 100,000 requests/day limit. The Pages Functions billing model introduced on November 15th allows your traffic to scale without limitation.
+
+## How it works 
+Pages Functions billing will be merged with the existing Workers billing structure for Free, Paid, and Enterprise customers. Read below to learn what this means for you as it relates to your Workers subscription plan: 
+
+{{<Aside type="note" header="Some title">}}
```
Comment (WalshyDev) [inline content/pages/platform/functions/billing.md]:
```suggestion
{{<Aside type="note">}}
```

File: content/pages/platform/functions/billing.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+# Pages Functions GA Billing 
+
+## About Pages Functions Billing 
+One of the key features of Pages Functions is limitless requests. In beta, Pages Functions had a 100,000 requests/day limit. The Pages Functions billing model introduced on November 15th allows your traffic to scale without limitation.
+
+## How it works 
+Pages Functions billing will be merged with the existing Workers billing structure for Free, Paid, and Enterprise customers. Read below to learn what this means for you as it relates to your Workers subscription plan: 
+
+{{<Aside type="note">}}
+If you had your Functions limit increased by the Pages team as part of the beta period, please note that this limit will no longer apply. Requests over 100k per day will be billed if on a paid/ENT plan or will not be served if on a free plan.
```
Comment (WalshyDev) [inline content/pages/platform/functions/billing.md]:
```suggestion
If you had your Functions limit increased by the Pages team as part of the beta period, please note that this limit will no longer apply. Requests over 100k per day will be billed if on Workers Paid (or Worker contract) or will not be served if on Workers Free. 
```

File: content/pages/platform/functions/billing.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+# Pages Functions GA Billing 
+
+## About Pages Functions Billing 
+One of the key features of Pages Functions is limitless requests. In beta, Pages Functions had a 100,000 requests/day limit. The Pages Functions billing model introduced on November 15th allows your traffic to scale without limitation.
+
+## How it works 
+Pages Functions billing will be merged with the existing Workers billing structure for Free, Paid, and Enterprise customers. Read below to learn what this means for you as it relates to your Workers subscription plan: 
+
+{{<Aside type="note">}}
+If you had your Functions limit increased by the Pages team as part of the beta period, please note that this limit will no longer apply. Requests over 100k per day will be billed if on Workers Paid (or Worker contract) or will not be served if on Workers Free.
```
Comment (WalshyDev) [inline content/pages/platform/functions/billing.md]:
```suggestion
If you had your Functions limit increased by the Pages team as part of the beta period, please note that this raised limit will no longer apply. If on Workers free, Functions invocations over 100k per day will not be run. If on Workers Paid (or Workers contract) all requests will be billed.
```

File: content/pages/platform/functions/billing.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+* Today, we recommend that anyone currently using over (or near) 100,000 requests on Pages Functions, Workers or both combined should upgrade to Workers Paid. 
+
+### Enterprise 
+* Your Workers contract allocation will now be shared between Workers and Functions.
+* If you currently have a Workers contract and need to make an addition to your contract, you can reach out to your Customer Service Manager (CSM) to make this change. 
+* If you do not have a Workers contract, you can reach out to your Customer Service Manager (CSM) to make this change. 
+
+{{<Aside type="note" header="Some title">}}
+If you had your Functions limit increased by the Pages team as part of the beta period, please note that this limit will no longer apply. Requests over 100k per day will be billed if on a paid tier or will not be served if on a free tier
+{{</Aside>}}
```
Comment (WalshyDev) [inline content/pages/platform/functions/billing.md]:
```suggestion
```

**Comment by WalshyDev** [comment]
🚢

**Comment by isaac-mcfadyen** [comment]
LGTM! 👍

**Pull request merged**